lib/portlayer/exec/base.go

```diff
@@ -36,6 +37,9 @@ import (
 	log "github.com/Sirupsen/logrus"
 )
 
+// The full string should be "The attempted operation cannot be performed in the current state (Powered off)"
+const opNotPerformedPoweredOff = "current state (Powered off)"
```

Comment:
You should use the full string. Is there no way to check the actual type and content of the error?